diff --git a/TerraformWorkShop/Nomad/Jobs/postgresql/terraform.tfvars b/TerraformWorkShop/Nomad/Jobs/postgresql/terraform.tfvars
index 8bb829b3..8d06a571 100644
--- a/TerraformWorkShop/Nomad/Jobs/postgresql/terraform.tfvars
+++ b/TerraformWorkShop/Nomad/Jobs/postgresql/terraform.tfvars
@@ -3,29 +3,29 @@ prov_nomad = {
   skip_verify = true
 }
 
-dynamic_host_volumes = [
-  {
-    name = "hvol-test-db"
-    constraint = [
-      {
-        attribute = "$${attr.unique.hostname}"
-        operator  = "=="
-        value     = "day3"
-      }
-    ]
-    capability = {
-      access_mode = "single-node-writer"
-    }
-    plugin_id = "mkdir"
-    parameters = {
-      uid = 26
-      gid = 26
-    }
-  },
-]
+# dynamic_host_volumes = [
+#   {
+#     name = "hvol-test-db"
+#     constraint = [
+#       {
+#         attribute = "$${attr.unique.hostname}"
+#         operator  = "=="
+#         value     = "day3"
+#       }
+#     ]
+#     capability = {
+#       access_mode = "single-node-writer"
+#     }
+#     plugin_id = "mkdir"
+#     parameters = {
+#       uid = 26
+#       gid = 26
+#     }
+#   },
+# ]
 
-jobs = [
-  {
-    path = "./attachments/test-db.nomad.hcl"
-  },
-]
+# jobs = [
+#   {
+#     path = "./attachments/test-db.nomad.hcl"
+#   },
+# ]
